integrations/adobe-analytics/lib/index.js

```diff
@@ -1497,23 +1497,30 @@ function createStandardVideoMetadata(track, mediaObj) {
   var props = track.properties();
   var metaKeys = videoAnalytics.MediaHeartbeat.VideoMetadataKeys;
   var stdVidMeta = {};
-  var segAdbMap = {
-    program: metaKeys.SHOW,
-    season: metaKeys.SEASON,
-    episode: metaKeys.EPISODE,
-    assetId: metaKeys.ASSET_ID,
-    contentAssetId: metaKeys.ASSET_ID,
-    genre: metaKeys.GENRE,
-    airdate: metaKeys.FIRST_AIR_DATE,
-    publisher: metaKeys.ORIGINATOR,
-    channel: metaKeys.NETWORK,
-    rating: metaKeys.RATING
-  };
+  var adbSegMap = {}
+  adbSegMap[metaKeys.SHOW] = ["program"],
+  adbSegMap[metaKeys.SEASON] = ["season"],
+  adbSegMap[metaKeys.EPISODE] = ["episode"],
+  adbSegMap[metaKeys.ASSET_ID] = [ "content_asset_id", "contentAssetId"],
```

Comment:
Do this logic prioritize `content_asset_id` or `contentAssetId` first? I think we want to always prioritize `contentAssetId` as to not break any existing implementation (i.e. if the customer happens to have both snake and camel case in their event - unlikely but good to be safe). Fallback on `content_asset_id`.

Reply:
It is currently prioritizing `content_asset_id`. I will make the correction.

Reply:
The test below makes me think that maybe these got swapped. Can we swap them back @rhall-twilio? Otherwise all looks good to me! Will wait for local testing to be complete and review from another eng before approving. Thanks!

Reply:
So fast at responding @rhall-twilio 😂

Reply:
This is now resolved.